adding_species/contributing.ipynb

```diff
@@ -556,15 +588,285 @@
     "from matplotlib import pyplot as plt\n",
     "\n",
     "x = process_sp(\"GAS.meru_mela.sfs.sp.summary\",1/11)\n",
+    "#x[0] are the times, x[1] are the popnsizes\n",
     "plt.plot(x[0],x[1])\n",
     "plt.xlabel(\"time\")\n",
     "plt.ylabel(\"N\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "revised-culture",
+   "metadata": {},
+   "source": [
+    "### add demography.py\n",
+    "To add species specific demographic models, we need to add a file called `demographic_models.py` into our species directory.\n",
+    "We can go look at the Arabidopsis demographic model entries for some simple\n",
+    "population size change histories. We can find them at `stdpopsim/catalog/AraTha/demographic_models.py`\n",
+    "\n",
+    "For `AnoGam` I'll create an analogous file called\n",
+    "`stdpopsim/catalog/AnoGam/demographic_models.py`\n",
+    "with the following header."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "living-rochester",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import msprime\n",
+    "import numpy as np\n",
+    "import stdpopsim\n",
+    "\n",
+    "_species = stdpopsim.get_species(\"AnoGam\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "monetary-mexican",
+   "metadata": {},
+   "source": [
+    "then into here I will add individual models. The model above could be entered as the following"
```

Comment:
This I think will require a good deal of explanation, or at least comments in the Python code.